doc/glossary.rst

```diff
@@ -15,6 +15,9 @@
                 Ceph's default and recommended storage back end.
 
 	Ceph
+                Ceph is a distributed network storage and file system with
+                distributed metadata management and POSIX semantics.
```

Comment:
@colemitchell: Maybe it would be better if it said "Ceph is a distributed network storage system. It offers block storage, object storage, and file storage. Ceph also provides metadata management, and POSIX semantics."

The text here (in the commit, not in the paragraph above in this comment) was taken verbatim from the "Architecture" document in the argonaut release branch of the git repo (that means "the first official release of Ceph").

I'll send an email to the developer list, asking for a better and more current definition of Ceph. For the time being, though, I'm going to merge this PR so that we can close another gap in the glossary.

I take your point, though. "distributed network storage and file system" is a phrase that does not scan at all, as far as I can surmise, for people who don't already know what these things are.